internal/v2/api/client_test.go

```diff
@@ -0,0 +1,37 @@
+package api_test
```

Comment:
basic sanity test - to verify that the mock works (and serves as an example as well).